src/flagser_bindings.cpp

```diff
@@ -122,8 +126,9 @@ PYBIND11_MODULE(flagser_pybind, m) {
     std::cout.rdbuf(cout_buff);
 
     // Remove generate output file
```

Comment:
"generated"?

Reply:
The comment is an old one, but needs a fix, let me also add it. Nice spotted.